ext/pdo/pdo_stmt.c

```diff
@@ -1451,12 +1466,16 @@ static void register_bound_param(INTERNAL_FUNCTION_PARAMETERS, int is_param) /*
 	param.param_type = (int) param_type;
 
 	if (param.name) {
+		if (ZSTR_LEN(param.name) == 0) {
```

Comment:
Was this previously enforced somewhere else? Makes sense though.

Reply:
It was not, but as with other things in PDO this seems like an edge case which was forgotten